rs/consensus/src/consensus/purger.rs

```diff
@@ -781,10 +781,10 @@ mod tests {
                         non_finalized_notarization_2
                     )),
                     ChangeAction::RemoveFromValidated(ConsensusMessage::BlockProposal(
-                        non_finalized_block_proposal_2_0
```

Comment:
@kpop-dfinity @dist1ll @eichhorl the expectation here is changed 
PTAL (here it is just a reordering change so most likely things are fine).